portal/tasks.py

```diff
@@ -375,3 +376,11 @@ def token_watchdog(**kwargs):
     if error_emails:
         return '\nUnable to reach recipient(s): {}'.format(
             ', '.join(error_emails))
+
+
+@celery.task
+def celery_beat_health_check():
+    """Pings the celery beat helath check API for monitoring"""
+    return requests.get(
```

Comment:
I might persist this through redis, worried how global variables behave in a multi-threaded environment. eg:
```
redis.from_url(app.config['REDIS_URL']).setex(
    'celerybeat_ping',
    'ping', 
    current_app.config['CELERY_BEAT_PING_INTERVAL'],
)
```

Reply:
https://stackoverflow.com/questions/32815451/are-global-variables-thread-safe-in-flask-how-do-i-share-data-between-requests